conans/client/loader.py

```diff
@@ -43,7 +44,7 @@ def invalidate_caches(self):
         self.cached_conanfiles = {}
         self._python_requires.invalidate_caches()
 
-    def load_class(self, conanfile_path, lock_python_requires=None):
+    def load_class(self, conanfile_path, lock_python_requires=None, env=dict()):
```

Comment:
default values to methods can NOT be objects, as they are reused among calls, and that is very dangerous and leads to errors difficult to debug

Reply:
changed to use `None` by default